lib/import.js

```diff
@@ -60,6 +67,25 @@ function parseText(text) {
   return partials;
 }
 
+function reformatToPartials(matlabFormatObj) {
```

Comment:
Maybe the prefix `re` is unnecessary since the word `format` already means to convert something from one structure to another, and also it depends on the caller of the function if something has already been called or not;

Also I personally prefer using `as` over `to`, following `format`:  

So how about `formatAsPartials`?